temporian/core/operators/test/test_unary.py

```diff
@@ -99,6 +99,53 @@ def test_correct_notnan(self) -> None:
         )
         assertOperatorResult(self, evset.notnan(), expected)
 
+    def test_round_single_feature(self):
+        evset = event_set(
+            timestamps=[1, 2, 3],
+            features={"f": [1.1, -2.5, -3.9]},
+        )
+        expected = event_set(
+            timestamps=[1, 2, 3],
+            features={"f": [1.0, -3.0, -4.0]},
+            same_sampling_as=evset,
+        )
+        assertOperatorResult(self, evset.round(), expected)
+        assertOperatorResult(self, round(evset), expected)  # __round__ magic
+
+    def test_round_multiple_features(self):
+        evset = event_set(
+            timestamps=[1, 2],
+            features={"a": [10.5, 11.7], "b": [1.2, 2.9]},
+        )
+        expected = event_set(
+            timestamps=[1, 2],
+            features={"a": [11, 12], "b": [1, 3]},
```

Comment:
I don't think these will pass, since a list of ints will yield an eventset with int features, which won't be equal to the float results of round(). Ensure your new tests are passing by running `bazel test //temporian/core/operators/test:test_unary --config=macos --test_output=errors` or `bazel test //temporian/core/operators/test:test_unary --config=linux --test_output=errors` depending on your OS